tests/script/github_action_helper.sh

```diff
@@ -58,13 +58,23 @@ test_flex_migration() {
   go build main.go
   TOOLBOX_POD=$(kubectl -n rook-ceph get pod -l app=rook-ceph-tools -o jsonpath='{.items[*].metadata.name}')
   kubectl -n rook-ceph cp main "$TOOLBOX_POD":/root/
-  kubectl -n rook-ceph exec -it "$TOOLBOX_POD" -- sh -c "cd root/ && ./main flexToCSI --sourcestoraageclass=rook-ceph-block --destinationstorageclass=csi-rook-ceph-block"
+  kubectl -n rook-ceph exec -it "$TOOLBOX_POD" -- sh -c "cd root/ && ./main --sourcestoraageclass=rook-ceph-block --destinationstorageclass=csi-rook-ceph-block"
+  exit_code_of_last_command=$?
+  if [ $exit_code_of_last_command -ne 0 ]; then
+    echo "Exit code migration command is non-zero. Migration failed"
```

Comment:
```suggestion
    echo "Migration failed"
```

Reply:
this is just for CI and log will help debug.

Reply:
Okay. In that case, let's keep the message: "Migration failed: exit code <record the exit code here>" . This will also help us know what is the exit code, in place of just saying it is non-zero. Wdyt?

Reply:
we can add the exit code but my question is how to exit code will help in debugging (apart or 0 or non-zeros).  My thought while adding is was is just to fail the test if the exit code is not zeros and logs from code will help in debugging.

Reply:
anyway, I just added it. Thanks